### PR TITLE
DDT-504-accept-editor-dialog

### PR DESCRIPTION
**Pull request by Abdiiir**
- Accept/Reject dialog box

### CODE REVIEW

File: ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      timestamp.eventLocation = null;
+      if (this.locationName) {
+        timestamp.eventLocation = new class implements EventLocation {
+          locationName: string
+        }
+        timestamp.eventLocation.locationName = this.locationName;
+      }
+    }
+
+    if (this.vesselPosition) { // If false older value is inhereted
```
Comment (nt-as) [inline ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts:151]: vesselPositionLabel?

Comment (Abdiiir) [reply ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts]: Resolved :)

File: ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+  ngOnInit(): void {
+    this.delayCodeService.getDelayCodes().subscribe(delayCodes => {
+      this.delayCodes = delayCodes;
+      this.updateDelayCodeOptions()
+    });
+    this.timestamps = this.config.data.timestamps;
+    this.transportCall = this.config.data.transportCall;
+    this.respondingToTimestamp = this.config.data.respondingToTimestamp;
+    console.log(this.respondingToTimestamp);
```
Comment (nt-as) [inline ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts:82]: left over debugging.

File: ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+</div>
+
+<div class="p-grid">
+  <div class="p-col-8">
+  </div>
+  <div class="p-col-4 footer_option" style="justify-content: flex-end">
+    <p-button (onClick)="close()" class="footer_option__buttons" icon="pi pi-times-circle"
+      label="{{'general.close.label' | translate}}" iconPos="left">
+    </p-button>
+    <p-button (onClick)="savePortcallTimestamp(respondingToTimestamp)" class="footer_option__buttons"
```
Comment (nt-as) [inline ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.html:134]: does not exist anymore.

Comment (nt-as) [reply ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.html:134]: If it is `responseTimestamp`, please remove the argument and just `this.responseTimestamp` inside `savePortcallTimestamp`.

**Comment by AndreasHoff** [comment]
Looks good to me, and testing worked fine :)

File: ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts (hunk 2 above)
Comment (Abdiiir) [reply ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts]: Done 👍

**Review by nt-as — APPROVED** [review]

**Review by AndreasHoff — COMMENTED** [review]
Looks good :)

**Pull request merged**